models/standardized_models/stripe__line_item_enhanced.sql

```diff
@@ -0,0 +1,294 @@
+{{ config(
+    enabled=(
+        var('stripe__standardized_billing_model_enabled', False) and 
+        (var('stripe__using_invoices', True) and var('stripe__standardized_billing_model_enabled', True))
```

Comment:
```suggestion
        (var('stripe__using_invoices', True) and var('stripe__standardized_billing_model_enabled', False))
```
This should be disabled by default.